RoadMap

```diff
@@ -0,0 +1,25 @@
+StartDate - 15th Aug 2019
+EndDate - By End of 2019
+
+List of Data Structures & Algorithms : 4 months ( 18 weeks) : 
+
+Array (2 weeks)
```

Comment:
Thinking how will we ensure if the questions added are enough. 
Should we decide a benchmark - 1 week topic means 10 questions?
2 week topic means 20 questions.

And then add all of those questions before the start of that week

Reply:
We shoudnt limit ourselves to the number of questions. We will try to cover all variety of questions in geeksforgeeks. the mentioned time should be enough to cover those(we can limit the number of problems to be coded, by skipping the similarly solved questions)

Reply:
lets try to cover as many questions as possible